cdap-app-templates/cdap-etl/cdap-etl-api/pom.xml

```diff
@@ -36,6 +36,10 @@
       <artifactId>cdap-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
Don't add guava to API package.

Reply:
Changed scope to `provided` as discussed.